.travis.yml

```diff
@@ -31,6 +31,22 @@ env:
 #   https://docs.travis-ci.com/user/osx-ci-environment/
 matrix:
   include:
+    - os: windows
+      language: shell
+      before_install:
+        - choco install python2 -x86
+        - ln -s /c/Python27/python.exe /c/Python27/python2.7.exe
+      env:
+        - PATH="/c/Python27:/c/Python27/Scripts:$PATH"
+        - TEST_SUITE="$PYTEST"
+    - os: windows
+      language: shell
+      before_install:
+        - choco install python2 -x86
+        - ln -s /c/Python27/python.exe /c/Python27/python2.7.exe
```

Comment:
We need to create a soft link pointing to `/c/Python27/python.exe` as during the configuration it will point to this executable file.